new.go

```diff
@@ -114,6 +116,17 @@ func imageManifestAndConfig(ref types.ImageReference, systemContext *types.Syste
 	return nil, nil, nil
 }
 
+func getFailureCause(err, defaultError error) error {
```

Comment:
Is this a function we could (should?) put into buildah/util.go?  The same file that has ResolveName in it, this might be useful in other situations me thinks.

Reply:
Yeah, why not.